src/accelerate/utils/memory.py

```diff
@@ -61,6 +62,8 @@ def clear_device_cache(garbage_collection=False):
         torch.mps.empty_cache()
     elif is_cuda_available():
         torch.cuda.empty_cache()
+    elif is_hpu_available():
+        pass
```

Comment:
Perhaps we should do a `logger.info()` here so people know that Gaudi is built different :wink:

Reply:
added